12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,34 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+  leo@leo-VirtualBox:~/lab$ ls *.[a,e,i,o,u]
```

Comment:
when we say "files" we usually mean files & directories... this is because file **are** directories :) actually the zen of  unix is that (almost) everything has a file interface, even devices! (check /dev if you don't believe me ;) )   
 
also note that extensions in Unix are **completely** optional... for example ```/etc/passwd``` is a text file. also this command  won't really work..  you need to do this: ```*.*[aeiou]``` in order to find files (and directories!) that contain dot in their name..